modules/inventory/server.lua

```diff
@@ -1342,6 +1342,37 @@ function Inventory.RemoveItem(inv, item, count, metadata, slot, ignoreTotal)
 end
 exports('RemoveItem', Inventory.RemoveItem)
 
+---@param inv inventory
+---@param items table
+---@param metadata? table | string
+function Inventory.CanCarryItems(inv, items, metadata)
+    inv = Inventory(inv) --[[@as OxInventory]]
+
+    local totalWeight = 0
+    for itemName, itemCount in pairs(totalItems) do
+        local item = Items(itemName)
+        if not item then goto skip end
+        
+        local itemSlots, _, emptySlots = Inventory.GetItemSlots(inv, item, type(metadata) == 'table' and metadata or { type = metadata or nil })
+        if not itemSlots or not next(itemSlots) and emptySlots == 0 or not item.stack and emptySlots < itemCount then return end
+
+        local weight = metadata and metadata.weight or item.weight
+        if weight == 0 then goto skip end
+
+        local itemsWeight = (weight * itemCount)
+        local newWeight = inv.weight + 
+        totalWeight += itemsWeight
```

Comment:
huh

Reply:
Should’ve been `local newWeight = inv.weight + itemsWeight` must’ve accidentally removed that.